profile/models.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import unicode_literals
```

Comment:
this is not needed, in python3 all strings are unicode by default

Reply:
You are right.